tasks/main.yml

```diff
@@ -28,6 +28,32 @@
   shell: >
     mv /usr/local/{{ sonar_version_directory }} /usr/local/sonar
     creates=/usr/local/sonar/COPYING
+  when: sonar_install_method|lower == 'move'
```

Comment:
Can you leave out the `|lower`? I think it's enough to have the docs mention move/copy/link in lower case, and presume that the user would follow that convention (instead of capitalizing).